datalad/distribution/get.py

```diff
@@ -685,6 +685,54 @@ class Get(Interface):
     and possibly assigned remote priorities), unless a specific source is
     specified.
 
+    *Getting subdatasets*
+
+    Just a DataLad supports getting file content from more than one location,
+    the same supported for subdatasets, including a ranking of individual
+    sources for prioritization.
+
+    The following locations candidates are considered. For each candidate a
+    cost is given in parenthesis, lower values indicate higher cost:
```

Comment:
maybe add something along the lines of "and thus lower priority"?

Reply:
Yeah, it is even more broken. Fixed.